example/docker-compose/multi-tenant/docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3"
```

Comment:
i'm always wary of adding new examples b/c they require manual testing/maintenance occassionally.

wdyt about coming this with the otel-collector multitenant example?

Reply:
agree, it adds the maintaince overhead but I would prefer to keep it out because otel-collector multitenant is an example setup of using otel collector to send data to a multi-tenant tempo setup, and this is an example of multi-tenant search.

they are same in the spirit I am worrited that if we merge, It might confused folks who are using these examples as reference config for standing up tempo...

so I am leaning on the side of keeping otel-collector multitenant as simple as possible and keep this out of otel-collector multitenant.